ooniprobe/Model/Database/Measurement.h

```diff
@@ -38,5 +38,5 @@
 -(void)save;
 -(void)setReRun;
 -(void)deleteObject;
--(void)getExplorerUrl:(void (^)(NSString *))measurement_url;
+-(void)getExplorerUrlSuccessCallback:(void (^)(NSString*))success errorCallback:(void (^)(NSError*))error;
```

Comment:
I find the name `getExplorerIUrlSuccessCallback` a bit verbose. How about (considering usage):

```ObjectiveC
[measurement getExplorerUrl:^(NSString *url) {
  // success case
} otherwise:^(NSError *error) {
  // failure case
}];
```

?